src/style.css

```diff
@@ -1 +1,83 @@
 /* Styles go here */
+@font-face {
+  font-family: Roboto;
+  src: url("fonts/Roboto/Roboto-Medium.ttf");
+}
+
+:root {
+  --horizontal-padding: 50px;
+  --header-height: 60px;
+  --underline-height: 4px;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+
+  width: calc(100% - 2 * var(--horizontal-padding));
```

Comment:
You dont need to calculate width as this header is alreadi taking 100% width and padding is included